plugins/rake-fast/rake-fast.plugin.zsh

```diff
@@ -8,7 +8,11 @@ _rake_refresh () {
 }
 
 _rake_does_task_list_need_generating () {
-  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]]
+  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]] || (_is_rails_app && [[ lib/tasks -nt .rake_tasks ]])
```

Comment:
`lib/tasks` is a folder right? Have you tested this? It's possible that the folder timestamp doesn't change when adding or modifying tasks inside of it.